ibm/resource_ibm_scc_account_settings.go

```diff
@@ -0,0 +1,124 @@
+	return &schema.Resource{
+		CreateContext: resourceIbmSccAccountSettingsUpdate,
+		ReadContext:   resourceIbmSccAccountSettingsRead,
+		UpdateContext: resourceIbmSccAccountSettingsUpdate,
+		DeleteContext: resourceIbmSccAccountSettingsDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"location_id": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: InvokeValidator("ibm_scc_account_settings", "location_id"),
+				Description:  "The programatic ID of the location that you want to work in.",
+			},
+			"modified": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The time that the location was last updated.",
+			},
+			"cleanup_inactive_locations": &schema.Schema{
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Used to determine when to delete data in an inactive location.",
+			},
+		},
+	}
+}
+
+func resourceIbmSccAccountSettingsRead(context context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	adminServiceClient, err := meta.(ClientSession).AdminServiceApiV1()
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	getAccountSettingsOption := &adminserviceapiv1.GetSettingsOptions{}
+
+	userDetails, err := meta.(ClientSession).BluemixUserDetails()
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	accountID := userDetails.userAccount
+
+	getAccountSettingsOption.SetAccountID(accountID)
+
+	accountSettings, response, err := adminServiceClient.GetSettingsWithContext(context, getAccountSettingsOption)
+	if err != nil {
+		if response != nil && response.StatusCode == 404 {
+			return nil
+		}
+		log.Printf("[DEBUG] GetSettingsWithContext failed %s\n%s", err, response)
+		return diag.FromErr(fmt.Errorf("GetSettingsWithContext failed %s\n%s", err, response))
+	}
+
+	d.SetId(*accountSettings.Location.ID)
+
+	return nil
+
+}
+
+func resourceIbmSccAccountSettingsUpdate(context context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	adminServiceClient, err := meta.(ClientSession).AdminServiceApiV1()
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	userDetails, err := meta.(ClientSession).BluemixUserDetails()
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	accountID := userDetails.userAccount
+
+	locationID := d.Get("location_id").(string)
+	updateAccountSettingsOption := &adminserviceapiv1.PatchAccountSettingsOptions{}
+	updateAccountSettingsOption.SetAccountID(accountID)
+	updateAccountSettingsOption.SetLocation(&adminserviceapiv1.LocationID{
+		ID: core.StringPtr(locationID),
+	})
+
+	_, response, err := adminServiceClient.PatchAccountSettingsWithContext(context, updateAccountSettingsOption)
+	if err != nil {
+		log.Printf("[DEBUG] PatchAccountSettingsWithContext failed %s\n%s", err, response)
+		return diag.FromErr(fmt.Errorf("PatchAccountSettingsWithContext failed %s\n%s", err, response))
+	}
+
```

Comment:
Set the id here d.SetID(XXXXXX)...
and in Read method set all other attributes 
modified, cleanup_inactive_locations